apps/aehttp/src/aehttp_dispatch_int.erl

```diff
@@ -32,8 +32,12 @@
                    ) -> {Status :: cowboy:http_status(), Headers :: list(), Body :: map()}.
 
 handle_request(OperationID, Req, Context) ->
-    jobs:run(http_update,
-             fun() -> handle_request_(OperationID, Req, Context) end).
+    try aec_jobs_queues:run(http_update,
+                            fun() -> handle_request_(OperationID, Req, Context) end)
+    catch
+        error:{rejected, _} ->
+            {503, [], #{reason => <<"Temporary overload">>}}
```

Comment:
validation of this will fail in aechttp_api_handler line 56 making it throw an exception instead of replying the request

Reply:
proposal is to not validate when we return a 500 code

Reply:
Do you suggest that I should remove the 503s from swagger?